hosts.go

```diff
@@ -149,6 +149,19 @@ func (h *Hosts) Add(ip string, hosts ...string) error {
 		return fmt.Errorf("%q is an invalid IP address", ip)
 	}
 
+	// remove hosts from other ips if it already exists
+	for _, host := range hosts {
+		for _, p := range h.getHostPositions(host) {
+			if h.Lines[p].IP == ip {
+				break
```

Comment:
should this be a continue to next iteration instead? If the first line of the hosts file has the desired IP but the next line does not, then it won't catch it

Reply:
ah yes, continue... good catch

Reply:
still passed tests tho... scary :D